src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react"
```

Comment:
If you're gonna use `react-router` prefer using `<Outlet />` component instead 
https://reactrouter.com/docs/en/v6/components/outlet